### PR TITLE
refactor Upload only option

### PR DESCRIPTION
**Pull request by vrulevskyi**
Modifies https://github.com/jenkinsci/fortify-plugin/pull/25.

### CODE REVIEW

File: src/main/webapp/help-uploadOnly.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!--
+    (c) Copyright 2019 Micro Focus or one of its affiliates.
```
Comment (tps193) [inline src/main/webapp/help-uploadOnly.html:2]: Should it be 2020 copyright?

Comment (vrulevskyi) [reply src/main/webapp/help-uploadOnly.html]: Fixed for all ui helpers.

**Review by tps193 — APPROVED** [review]
Seems ok for me

**Comment by vrulevskyi** [comment]
Merged.

**Pull request merged**